NATIVE_SPEND.md

```diff
@@ -55,4 +55,80 @@ JWT need to be signed by server side where private key is secure.
         } catch (TaskFailedException e) {
             e.printStackTrace();
         }
+```
+
+# Add spend offer opportunity to marketplace
+1. Add a NativeOfferCallback to be triggered when the user clicks on your offer in Kin Marketplace.
+```java
+        try {
+            Kin.start(getApplicationContext(), signInData)
+            .addNativeOfferCallback(new Callback<NativeSpendOffer>() {
```

Comment:
can we separated the addNativeOfferCallback from Kin.start? should we add removeNativeOfferCallback()?

Reply:
why would we want to separate it?
it's clearer like that for devs that they should add callback before adding offers, also it safer.

Reply:
Not sure if we need remove method.

Reply:
What if Kin.start should be called as soon as possible but developer might want to add NativeOfferCallback just before launching the marketplace and remove it when it closed - not sure we need to change it now but we might need to

Reply:
👍